packages/interactive_media_ads/README.md

```diff
@@ -13,7 +13,11 @@ a separate video player positioned on top of the app's content video player.
 |-------------|---------|-------|
 | **Support** | SDK 21+ | 12.0+ |
 
-**This package is still in development.**
+**NOTE:**
+* The initial release for this package supports linear pre-roll video ads on iOS and Android
+* platforms. Please view/report any issues [here](https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+interactive_media_ads%22).
```

Comment:
What's the reason for the issue link? We have the metadata set that causes `pub.dev` to put an issues link in the sidebar, and that's where we expect clients to be seeing the README.

Reply:
Ah, that's right. That was my suggestion. I forgot that is already included in the pubspec. I removed it from here. 

cc @harold1208 to confirm this is fine as well

Reply:
Confirming this is fine, thanks!